reference/ToDo/src/ToDo/Views/HomePage.xaml

```diff
@@ -1,143 +1,165 @@
-							 ExpandedModeThresholdWidth="0">
+    <Style x:Key="TaskListPathIconStyle"
+           TargetType="PathIcon">
+      <Setter Property="Foreground"
+              Value="{ThemeResource PrimaryVariantDarkBrush}" />
+      <Setter Property="Data"
+              Value="{StaticResource Icon_House_Siding}" />
+    </Style>
 
-			<!-- Header: Profile (Setting button) + Search button -->
-			<muxc:NavigationView.PaneHeader>
-				<Grid Grid.Row="0"
-					  Padding="16,8"
-					  ColumnSpacing="8"
-					  ColumnDefinitions="*,Auto">
+    <Style x:Key="ImportantTaskListPathIconStyle"
+           TargetType="PathIcon">
+      <Setter Property="Foreground"
+              Value="{ThemeResource ErrorBrush}" />
+      <Setter Property="Data"
+              Value="{StaticResource Icon_Star_Rate}" />
+    </Style>
 
-					<Button Grid.Column="0"
-							uen:Navigation.Request="!Settings"
-							BorderThickness="0"
-							HorizontalAlignment="Stretch"
-							HorizontalContentAlignment="Stretch"
-							Style="{StaticResource OutlinedButtonStyle}">
-						<Grid ColumnDefinitions="Auto,*"
-							  ColumnSpacing="8">
-							<PersonPicture Grid.Column="0"
-								   DisplayName="{Binding CurrentUser.Name}"
-								   ProfilePicture="{Binding ProfilePicture, Converter={StaticResource  BitmapSourceConverter}}"
-								   Width="40"
-								   Height="40"
-										   VerticalAlignment="Center" />
-							<TextBlock Grid.Column="1"
-									   Text="{Binding CurrentUser.Name}"
-									   VerticalAlignment="Center"
-									   Style="{StaticResource TitleMedium}" />
-						</Grid>
-					</Button>
+    <converters:TaskListToValueConverter x:Key="TaskListPathIconStyleConverter"
+                                         ImportantValue="{StaticResource ImportantTaskListPathIconStyle}"
+                                         TasksValue="{StaticResource TaskListPathIconStyle}"
+                                         DefaultValue="{StaticResource DefaultTaskListPathIconStyle}" />
+  </Page.Resources>
+  <Grid uut:VisibleBoundsPadding.PaddingMask="All">
+    <VisualStateManager.VisualStateGroups>
+      <VisualStateGroup>
+        <VisualState x:Name="Wide">
+          <VisualState.StateTriggers>
+            <AdaptiveTrigger MinWindowWidth="{StaticResource WideMinWindowWidth}" />
+          </VisualState.StateTriggers>
+          <VisualState.Setters>
+            <Setter Target="NavView.(uen:Navigation.Request)"
+                    Value="./TaskList" />
+          </VisualState.Setters>
+        </VisualState>
+        <VisualState x:Name="Narrow">
+          <VisualState.StateTriggers>
+            <AdaptiveTrigger MinWindowWidth="0" />
+          </VisualState.StateTriggers>
+          <VisualState.Setters>
+            <Setter Target="NavView.(uen:Navigation.Request)"
+                    Value="TaskList" />
+          </VisualState.Setters>
+        </VisualState>
+      </VisualStateGroup>
+    </VisualStateManager.VisualStateGroups>
```

Comment:
Another future consideration here is to use Responsive Extensions for this

Reply:
I will give it a try

Reply:
I switched to ResponsiveExtension when needed and used ResponsiveView for the WelcomePage